e2e/cypress/integration/multi_team_and_dm/dm_more_searching_from_page_2_spec.js

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @multi_team_and_dm
+
+describe('Multi Team and DM', () => {
+    let testChannel;
+    let testTeam;
+    let testUser;
+    let searchTerm;
+    const prefix = 'testuser';
+
+    before(() => {
+        // # Setup with the new team, channel and user
+        cy.apiInitSetup().then(({team, channel, user}) => {
+            testTeam = team;
+            testChannel = channel;
+            testUser = user;
+            searchTerm = user.username;
+
+            // # Create 52 users so the user must page forward in the dm list
+            Cypress._.times(52, () => {
+                cy.apiCreateUser({prefix: `${prefix}`}).then(() => {
+                    cy.apiAddUserToTeam(testTeam.id, user.id);
+                });
+            });
+
+            // # Login with testUser and visit test channel
+            cy.apiLogin(testUser);
+            cy.visit(`/${testTeam.name}/channels/${testChannel.name}`);
+        });
+    });
+
+    it('MM-T446 DM More... searching from page 2 of user list', () => {
+        // # Open the Direct Message modal
+        cy.findByLabelText('write a direct message').click();
+
+        // # Move to the next page of users
+        cy.findByText('Next').click().then(() => {
+            cy.findByText('Previous').should('exist');
```

Comment:
No need to next this.
```suggestion
        cy.findByText('Next').click()
        cy.findByText('Previous').should('exist');
```